docs/help_out.txt

```diff
@@ -124,6 +124,6 @@ GLOBAL OPTIONS:
    --storage.backends-to-enable value [ --storage.backends-to-enable value ]  Comma separated list of eigenDA backends to enable (e.g. V1,V2) (default: "V1") [$EIGENDA_PROXY_STORAGE_BACKENDS_TO_ENABLE]
    --storage.cache-targets value [ --storage.cache-targets value ]            List of caching targets to use fast reads from EigenDA. [$EIGENDA_PROXY_STORAGE_CACHE_TARGETS]
    --storage.concurrent-write-routines value                                  Number of threads spun-up for async secondary storage insertions. (<=0) denotes single threaded insertions where (>0) indicates decoupled writes. (default: 0) [$EIGENDA_PROXY_STORAGE_CONCURRENT_WRITE_THREADS]
-   --storage.disperse-to-v2                                                   Enable blob dispersal to EigenDA V2 protocol. (default: false) [$EIGENDA_PROXY_STORAGE_DISPERSE_TO_V2]
+   --storage.dispersal-backend value                                          Target EigenDA backend version for blob dispersal (e.g. V1 or V2). (default: "V1") [$EIGENDA_PROXY_STORAGE_DISPERSAL_BACKEND]
```

Comment:
I don't like "(e.g. V1 or V2)"
I think (?) there's a way to use an enum for allowed values in urfave and urfave would automatically list all the options here. Don't spend too much time on this but if its easy would be a nice future-compatible change

Reply:
I took a crack at this:

1. It's kinda ugly (if you're curious to see, I pushed it to a branch called `cli-enums`
2. I was struggling to get the help output to generate using the generic type

Given our current time crunch, I think it's best to more forward as is, since it's not proven super easy to implement, and I'm not yet 100% convinced it's actually better, considering the added complexity.

Reply:
LGTM def not a priority. Pretty sure op uses this in a few places, so we could also look at that for other examples if/when we decide this is worth doing (never is fine lol)